lib/config.js

```diff
@@ -110,12 +111,54 @@ module.exports = function (cfg) {
   }
 
   // --------------------------------------------------------------------------
-  // Settings: Available Selenium-ish options.
+  // Settings: Extract current desired setting path
   // --------------------------------------------------------------------------
-  if (!cfg.settings.default) {
-    cfg.settings.default = GLOBAL_DEFAULT;
+  var settingsPath = ROWDY_SETTINGS.replace(QUOTES_RE, "").split(".");
+  settingsPath = _.chain(settingsPath)
+    .map(function (v) { return v.trim(); })
+    .filter(_.identity)
+    .value();
+
+  // --------------------------------------------------------------------------
+  // Browser setting extensions (Guacamole)
+  // --------------------------------------------------------------------------
+  if (cfg.options.guacamole) {
+    // Lazy-require.
+    var guacamole = require("guacamole");
+
+    // SauceLabs.
+    if (settingsPath[0] === "sauceLabs") {
+      //
+      // TODO: Fix this path
```

Comment:
One tricky bit here:

In this code's context, here's what I get for the following two calls:

 * `process.cwd()`: `/Users/xxxxx/rowdy/examples/mocha/wd` (hence `../../../`, but this assumes a certain kind of tree in the user's setup)
 * `__dirname`: `/Users/xxxxx/rowdy/lib` (relative to included module, so clearly not appropriate..)

hmmm... :)